modules/db/src/main/java/org/mule/module/db/internal/config/domain/query/ParameterizedQueryTemplateFactoryBean.java

```diff
@@ -143,4 +147,10 @@ public boolean isSingleton()
     {
         return false;
     }
+
+    private String buildNotDefinedInParamErrorMessage(String name)
+    {
+        return "Parameter with name " + name + ", used in the query text, does not match any defined query parameter name.";
```

Comment:
Use String format and change the string to be like "Parameter with name 'xxx', used ..."